entity-event-generator-lambda/src/test/scala/uk/gov/nationalarchives/testUtils/LambdaSpec.scala

```diff
@@ -0,0 +1,129 @@
+package uk.gov.nationalarchives.testUtils
```

Comment:
How come this file has been moved to `testUtils`?

Reply:
Don't know. I was moving it to uk.gov.nationalarchives and went one too far. I've moved it into the parent directory.